modules/profile/files/builder/builder-do-update.sh

```diff
@@ -11,9 +11,7 @@ trap cleanup EXIT
 export NODE_ENV=production
 
 cd "$1"
-npm install
-npm update
-npm prune
+npm ci
```

Comment:
I'd recommend keeping the builder agnostic about whether a repo has a package-lock file.

```suggestion
if [ -e 'package-lock.json' ] || [ -e 'npm-shrinkwrap.json' ]; then
  npm ci
else
  npm install
fi
```

Reply:
One could make the argument it should throw an error if it doesn't, but I'm fine with this change.